apis/python/test/test_index.py

```diff
@@ -350,7 +350,46 @@ def test_ivf_pq_index(tmp_path):
         index, np.array([[2, 2, 2]], dtype=np.float32), 2, [[0, 3]], [[2, 1]]
     )
 
-    # TODO(paris): Add tests for consolidation once we enable it.
```

Comment:
Consolidation was added in https://github.com/TileDB-Inc/TileDB-Vector-Search/pull/447, here we add tests based on Vamana.